conda/recipes/legate-boost/conda_build_config.yaml

```
@@ -19,11 +19,11 @@ cuda_compiler:
 cuda11_compiler:
   - nvcc
 
-# match the legate-core / cunumeric pattern of publishing
+# match the legate / cunumeric pattern of publishing
 # CPU-only and GPU-only packages for every variant
 gpu_enabled:
   - true
   - false
 
 legate_version:
-  - =24.06.*,>=24.06.01
+  - ">=24.09.00.dev0"
```

```suggestion
  - "=24.09.*,>=24.09.00.dev0"
```

Similar comment here... I think we still want to stay within the `24.09` series of releases. Based on my experience with `legate` so far, it seems highly unlikely that a prospective `legate-boost=24.09.00` would be compatible with a future `cunumeric=24.12.00`.